gnomad_hail/utils/annotations.py

```diff
@@ -419,3 +425,503 @@ def annotate_freq(
+        )
+    )
+
+    sibs = sib_ht.sibs.collect()
+    sibs_type = hl.dtype(
+        "array<struct{sib1:int32,sib2:int32,sib1_is_female:bool,sib2_is_female:bool}>"
+    )
+    mt = mt.annotate_globals(**{"sibs": hl.literal(sibs, sibs_type)})
+    mt = mt.localize_entries("entry_structs", "columns")
+
+    mt = mt.annotate_globals(
+        **{
+            "columns": hl.map(
+                lambda i: hl.bind(
+                    lambda t: hl.struct(
+                        id=i,
+                        sib1=mt.columns[t.sib1],
+                        sib2=mt.columns[t.sib2],
+                        sib1_is_female=t.sib1_is_female,
+                        sib2_is_female=t.sib2_is_female
+                    ),
+                    mt.sibs[i]
+                ),
+                hl.range(0, len(sibs))
+            )
+        }
+    )
+
+    mt = mt.annotate(
+        **{
+            "entry_structs": hl.map(
+                lambda i: hl.bind(
+                    lambda t: hl.struct(
+                        sib1_entry=mt.entry_structs[t.sib1],
+                        sib2_entry=mt.entry_structs[t.sib2]
+                    ),
+                    mt.sibs[i]
+                ),
+                hl.range(0, len(sibs))
+            )
+        }
+    )
+
+    mt = mt._unlocalize_entries("entry_structs", "columns", ["id"])
+
+    return mt
+
+
+def default_generate_sib_stats(
+        mt: hl.MatrixTable,
+        relatedness_ht: hl.Table,
+        sex_ht: hl.Table,
+        i_col: str = 'i',
+        j_col: str = 'j',
+        relationship_col: str = 'relationship'
+) -> hl.expr.StructExpression:
+    """
+    This is meant as a default wrapper for `generate_sib_stats_expr`. It returns a hail table with counts of variants
+    shared by pairs of siblings in `relatedness_ht`.
+
+    This function takes a hail Table with a row for each pair of individuals i,j in the data that are related (it's OK to have unrelated samples too).
+    The `relationship_col` should be a column specifying the relationship between each two samples as defined by
+    the constants in `gnomad_hail.utils.relatedness`.
+
+    :param mt: Input Matrix table
+    :param relatedness_ht: Input relationship table
+    :param sex_ht: A Table containing sex information for the samples
+    :param i_col: Column containing the 1st sample of the pair in the relationship table
+    :param j_col: Column containing the 2nd sample of the pair in the relationship table
+    :param relationship_col: Column containing the relationship for the sample pair as defined in this module constants.
+    :return: A Table with the sibling shared variant counts
+    """
+    sex_ht = sex_ht.annotate(
+        is_female=hl.case()
+            .when(sex_ht.sex_karyotype == "XX", True)
+            .when(sex_ht.sex_karyotype == "XY", False)
+            .or_missing()
+    )
+
+    sib_ht = relatedness_ht.filter(relatedness_ht[relationship_col] == 'Siblings')
```

Comment:
change to constant from `relatedness.py`. Also, maybe add in the docstring that it's what this will be using?

Reply:
I just realized that this is in another PR! So I guess maybe add a comment to change it?